package.json

```diff
@@ -49,5 +29,25 @@
     "mocha": "^2.5.3",
     "npm-run-all": "^2.2.0",
     "webpack": "^1.13.1"
+  },
+  "homepage": "https://github.com/mozilla/page-metadata-parser#readme",
+  "keywords": [
+    "page",
+    "metadata",
+    "parser"
+  ],
+  "license": "MPL-2.0",
+  "main": "parser.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/page-metadata-parser.git"
+  },
+  "scripts": {
+    "cover": "cat ./coverage/lcov/lcov.info | coveralls",
+    "tdd": "npm run test:karma -- --no-single-run",
+    "test": "npm-run-all test:*",
+    "test:karma": "karma start",
+    "test:lint": "eslint .",
+    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec"
```

Comment:
Shortened to just `instanbul` instead of the more verbose `./node_modules/istanbul/lib/cli.js`.